### PR TITLE
Partners: voyageai - introducing the new voyageai models

### PR DESCRIPTION
**Pull request by fzowl**
**Description:**
Introducing new VoyageAI models

- [x] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

### CODE REVIEW

File: libs/partners/voyageai/langchain_voyageai/embeddings.py [hunk 1]
```diff
@@ -55,7 +55,15 @@ def default_values(cls, values: dict) -> Any:
...
-            values["batch_size"] = 72 if model in ["voyage-2", "voyage-02"] else 7
+            values["batch_size"] = (
+                72
+                if model in ["voyage-2", "voyage-02"]
+                else (
+                    30
+                    if model == "voyage-3-lite"
+                    else (10 if model == "voyage-3" else 7)
+                )
+            )
```
Comment (efriis) [inline libs/partners/voyageai/langchain_voyageai/embeddings.py:66]: any chance this is set in the sdk as well? if so can we just make this `None` and rely on the sdk for defaults?

**Review by efriis — COMMENTED** [review]
looks good! one question

File: libs/partners/voyageai/langchain_voyageai/embeddings.py (hunk 1 above)
Comment (fzowl) [reply libs/partners/voyageai/langchain_voyageai/embeddings.py:66]: @efriis Our SDK is thin and it is just a wrapper around the API. We let our clients to do the batching and do the batching in the integrations - wherever it is supported.

**Comment by efriis** [comment]
merging from #26907 because don't have write acccess to branch

**Pull request closed without merging**